src/OnePlusBot/Modules/ModMail.cs

```diff
@@ -21,7 +21,8 @@ public class ModMail : ModuleBase<SocketCommandContext>
           Command("reply", RunMode = RunMode.Async),
           Summary("Reply to a modmail thread"),
           RequireRole("staff"),
-          RequireModMailContext
+          RequireModMailContext,
+          Alias("r")
```

Comment:
I would personally move this and the second Alias attribute directly before the summary, for correctness.